README.md

```diff
@@ -46,6 +46,9 @@ export GITHUB_STATUS_CONTEXT="codebuild/pr"  # Tag associated with the status ch
 export GITHUB_BUILD_EVENTS="pr_state"  # Comma-separated string of build events, supports "pr_state" and "pr_comment"
 export GITHUB_BUILD_USERS=""  # Comma-separated string of GitHub users authorized to build from PR comments
 export GITHUB_BUILD_COMMENT="go codebuild go"  # Case-insensitive string that will start a build from a PR comment
+export CB_EXTERNAL_BUILDSPEC=false # If "false", webhook will not have CodeBuild submit source version in buildspec.yml request (useful if in 2 different repos or you want to use the most recent commit)
+export CB_GIT_REPO_ENV=REPO_TO_BUILD # Name of env var to pass to CodeBuild with repo triggering build
+export CB_GIT_REF_ENV=REF_TO_BUILD # Name of env var to pass to CodeBuild with PR# triggering build
```

Comment:
"Name of ENV variable that your CodeBuild project uses to know which git ref to checkout. For use with CB_EXTERNAL_BUILDSPEC."